packages/runtime/src/runtimes/react/components/LiveProvider.tsx

```diff
@@ -15,7 +15,12 @@ type Props = {
 export default function LiveProvider({ client, children, rootElements }: Props): JSX.Element {
   const runtime = useReactRuntime()
   const store = useMemo(
-    () => ReactPage.configureStore({ preloadedState: runtime.store.getState(), rootElements }),
+    () =>
+      ReactPage.configureStore({
+        name: 'Host store',
```

Comment:
Give each store a name so we can differentiate them more easily in the devtools UI.